src/app/components/ResultList.js

```diff
@@ -1,13 +1,16 @@
+// Utilities
 import React from 'react';
+import PropTypes from 'prop-types';
+
+// Components
 import Item from './Item';
 
-export default (props) => {
+export default function resultList(props) {
 
     const createItem = props.images.map((image) => {
         return (
             <figure
-                key={`_${image.id}`}
-                // key={`_${image.id}${Math.random().toString(36).slice(-5)}`}
+                key={`_${image.id}${Math.random().toString(36).slice(-5)}`}
```

Comment:
Create a custom method to generate a random ID as a helper or use libraries like `uuid`